src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -124,6 +123,12 @@ export const TradeInput = memo(() => {
   const sellAssetSearch = useModal('sellAssetSearch')
   const buyAsset = useAppSelector(selectBuyAsset)
   const sellAsset = useAppSelector(selectSellAsset)
+  const percentOptions = useMemo(() => {
+    if (!sellAsset?.assetId) return []
```

Comment:
Should be defined but in case it's not, we don't want to have a flash of max button